castReceiver-v7/receiverApp.html

```diff
@@ -0,0 +1,162 @@
+        <div class="app-status-overlay-loading-dot"></div>
+    </div>
+</div>
+
+<script type="text/javascript" src="https://bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer.js"></script>
+<script type="text/javascript" src="https://bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer-ui.js"></script>
+<script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.2/require.js" integrity="sha256-uvyqD81XGEqlTzEGkl+5L73IUlWTXtdLhfnUG5n3FbE="
+        crossorigin="anonymous"></script>
+<script type="text/javascript" src="https://bitmovin-a.akamaihd.net/bitmovin-player-cast/stable/7.6.0/bitmovinplayer-remotereceiver.js"></script>
+<script type="text/javascript">
+  var player, uimanager;
+
+  var hideStatusOverlay = () => {
+    document.getElementById('app-status-overlay').style.display = 'none';
+  };
+
+  window.onload = () => {
+    var conf = {
+      key: "YOUR-KEY",
+      tweaks: {
+        max_buffer_level: 20
+      },
+      style: {
+        ux: false
+      }
+    };
+
+    // Instantiate a Bitmovin Player
+    player = bitmovin.player('player-wrapper');
+
+    // Hide loading overlay when a source is loaded
+    player.addEventHandler(player.EVENT.ON_SOURCE_LOADED, hideStatusOverlay);
+
+    // Hide overlay when an error happens (avoids hidden errors during startup)
+    player.addEventHandler(player.EVENT.ON_ERROR, hideStatusOverlay);
+
+    player.addEventHandler(player.EVENT.ON_METADATA, (event) => {
+      switch (event.metadataType) {
+        case 'CAST':
+          switch (event.metadata.type) {
+            case 'customReceiverConfig':
+              var customReceiverConfig = event.metadata.data;
+
+              console.log('customReceiverConfig', customReceiverConfig);
+
+              if (customReceiverConfig.receiverStylesheetUrl != null) {
+                var head = document.getElementsByTagName('head')[0];
+                var link = document.createElement('link');
+                link.rel = 'stylesheet';
+                link.type = 'text/css';
+                link.href = customReceiverConfig.receiverStylesheetUrl;
+                head.appendChild(link);
+              }
+              break;
+          }
+          break;
+      }
+    });
+
+    player.setup(conf).then((player) => {
+      console.log('Successfully created Bitmovin player instance');
+
+      require(['src/remotecontrol/GoogleCastRemoteControlReceiver'], (gcr, wsr) => {
+        const googleCastRemoteControlReceiver = new gcr.GoogleCastRemoteControlReceiver(player);
+      });
+
+      // Load the built-in Bitmovin Cast Receiver UI
+      uimanager = bitmovin.playerui.UIManager.Factory.buildDefaultCastReceiverUI(player);
+    }).catch((error) => {
+      console.error('Error while creating Bitmovin player instance', error);
+    });
+
+    analytics.register(player);
```

Comment:
Shouldn't we register the `player` right after [L114](https://github.com/bitmovin/bitmovin-player-web-samples/pull/44/files#diff-707d17a22605ce993beaa590caa0b112R114) so analytics can also measure the startup performance?

Reply:
Actually, analytics isn't part of this sample since https://github.com/bitmovin/bitmovin-player-web-samples/pull/44/commits/19353cacb9b6e9d9513e3c57a049843d6439f1e3, this was a missed leftover.